index.js

```diff
@@ -16,3 +18,16 @@ mobileLinks.forEach((link) => {
     mobileMenu.classList.toggle('active');
   });
 });
+
+email.addEventListener('input', () => {
+  if (email.validity) {
+    email.setCustomValidity('');
+  }
+});
+
+form.addEventListener('submit', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    email.setCustomValidity('Email must be in lowercase');
+    e.preventDefault();
+  }
+});
```

Comment:
- [ ] Could you consider making sure that message appears close to the submit button or the input fields? and also consider using a `"click"` event to attach the function checking the email format.because that will help you to fulfill the below requirement 👇 
- > If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.